setup.py

```diff
@@ -33,6 +33,7 @@
     "pyyaml>=6.0",
     "docker>=6.1.3",
     "psutil>=5.9.5",
+    "requests>=2.31.0",
```

Comment:
`docker` already required this package, so it will already be present in any recent PyFluent environments